samples/errors6.go

```diff
@@ -0,0 +1,41 @@
+package main
```

Comment:
I've updated your sample with my suggestions and reformatted it a bit https://goplay.space/#F_EG5jM_fZ6

removed unnecessary blank line, moved inline, anonymous function to outer (easier to grok i think).
naked returns are discouraged unless you have good reason: https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
Thanks I've used it